distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -337,7 +337,36 @@
                                     <asyncDestroyOnShutdown>true</asyncDestroyOnShutdown>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>attach-ace-debugger</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>java</executable>
+                                    <arguments>
```

Comment:
❓Instead of using the `-jar` argument and hard-coding the path to the acdebugger jar file, could we use the [`<executableDependency/>` element](https://www.mojohaus.org/exec-maven-plugin/examples/example-exec-using-plugin-dependencies.html) instead?

Reply:
Problem with using that is you have to use the `java` goal instead of `exec` which runs it in the same vm as the tests and isn't async any more and blocks the tests from running

Reply:
I seem to can't get it to work with the `exec` goal, if i use the `java` goal it will run acdebugger in the same vm and block the tests because it's synchronous .